python/paddle/fluid/tests/unittests/ngraph/test_conv2d_ngraph_op.py

```diff
@@ -17,60 +17,5 @@
 import unittest
 from paddle.fluid.tests.unittests.test_conv2d_op import TestConv2dOp, TestWithPad, TestWithStride, TestWithGroup, TestWith1x1, TestWithInput1x1Filter1x1
 
-
-class TestNGRAPH(TestConv2dOp):
-    def setUp(self):
-        super(TestNGRAPH, self).setUp()
-        self._cpu_only = True
-
-    def init_kernel_type(self):
-        super(TestNGRAPH, self).init_kernel_type()
-
-
-class TestNGRAPHWithPad(TestWithPad):
-    def setUp(self):
-        super(TestNGRAPHWithPad, self).setUp()
-        self._cpu_only = True
-
-    def init_kernel_type(self):
-        super(TestNGRAPHWithPad, self).init_kernel_type()
-
-
-class TestNGRAPHWithStride(TestWithStride):
-    def setUp(self):
-        super(TestNGRAPHWithStride, self).setUp()
-        self._cpu_only = True
-
-    def init_kernel_type(self):
-        super(TestNGRAPHWithStride, self).init_kernel_type()
-
-
-class TestNGRAPHWithGroup(TestWithGroup):
-    def setUp(self):
-        super(TestNGRAPHWithGroup, self).setUp()
-        self._cpu_only = True
-
-    def init_kernel_type(self):
-        super(TestNGRAPHWithGroup, self).init_kernel_type()
-
-
-class TestNGRAPHWith1x1(TestWith1x1):
-    def setUp(self):
-        super(TestNGRAPHWith1x1, self).setUp()
-        self._cpu_only = True
-
-    def init_kernel_type(self):
-        super(TestNGRAPHWith1x1, self).init_kernel_type()
-
-
-class TestNGRAPHWithInput1x1Filter1x1(TestWithInput1x1Filter1x1):
-    def setUp(self):
-        super(TestNGRAPHWithInput1x1Filter1x1, self).setUp()
-        self._cpu_only = True
```

Comment:
Why do you remove all the unit-tests? remove `self._cpu_only = True` is enough?

Reply:
Here is my understanding. The testcases are imported to the file and they will be picked up by unittest. https://github.com/PaddlePaddle/Paddle/blob/b010354b12126dcbf3e0878380d7824970ae84f5/python/paddle/fluid/tests/unittests/ngraph/test_conv2d_ngraph_op.py#L18

Those testcases of TestConv2dOp, TestWithPad, TestWithStride, TestWithGroup, TestWith1x1, and TestWithInput1x1Filter1x1 will run and they will run through ngraph. 

The derived testcase has no change except it is set as cpu_only. Since this cpu_only setting is no longer needed, the test case is not needed.